Workbooks/WorkbooksMetadata.json

```diff
@@ -1142,15 +1142,15 @@
     "provider": "IntSights Cyber Intelligence"
   },
   {
-    "workbookKey": "DarktraceSummaryWorkbook",
+    "workbookKey": "AIADarktraceSummaryWorkbook",
     "logoFileName": "Darktrace.svg",
     "description": "A workbook containing relevant KQL queries to help you visualise the data in model breaches from the Darktrace Connector",
     "dataTypesDependencies": [ "CommonSecurityLog" ],
     "dataConnectorsDependencies": [],
```

Comment:
@SunRift : I see that the **DataConnectorDependencies** is not linked with DarkTrace Connector, is it done purposefully or is a miss?

Reply:
thanks, I had thought it would possibly restrict people from using the workbook, but on second thought, it shouldn't matter, because everyone with access to the workbook template will have access to the connector. 
 I will add it in.